server/src/server.rs

```diff
@@ -563,6 +438,12 @@ impl<HttpMiddleware, RpcMiddleware> Builder<HttpMiddleware, RpcMiddleware> {
 		self
 	}
 
+	/// Set the maximum number of connections allowed. Default is 1024.
```

Comment:
I don't understand why you changed these on the Builder?  It seems like you just moved the APIs around or am I missing something?

I would be easier if you just reverted these

Reply:
I just adjusted the order of the methods to make them consistent with the order of the fields in `ServerConfig` to avoid missing some methods.

Reply:
It's okay this time because it's rather small change but generally I prefer to keep each PR as precise/small as possible.